freqtrade/misc.py

```diff
@@ -110,6 +110,14 @@ def common_args_parser(description: str):
         type=str,
         metavar='PATH',
     )
+    parser.add_argument(
+        '-dd', '--datadir',
```

Comment:
Can you please change `-dd` so that we have only one character for single hyphen arguments.

> In the original Unix tradition, command-line options are single letters preceded by a single hyphen.

See http://www.faqs.org/docs/artu/ch10s05.html